src/docs/patterns/lists.mdx

```diff
@@ -1,56 +1,193 @@
-<DemoCard repository={'action-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Faction-list%2Faction-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/action-list/index.tsx&initialpath=action-list'} ionic reactNative />
+Commonly in PX Blue applications, list items also show some visual indication of the item's current status, such as online, offline, or an error condition. This is achieved by using avatars and status stripes.
 
-If there are actions that you would like to perform on individual list items, these can be included via icons or a 3-dot action menu (multiple actions) on each list item. Depending on the space available, you typically will want to use the action menu if you have more than three actions that can be taken. Global actions that affect the whole list (such as Add or Clear) should be placed in the list header.
+<ImageGrid
+    images={[IconAlignImage]}
+    regularWidth
+    caption={
+        'Whenever you have a list that may include both avatars and icons, make sure all the icons, with or without avatars, are center-aligned. If you are using interactive elements like checkboxes and radio buttons, they should also be center-aligned to accomodate ripples. Otherwise, all icons should be left-aligned.'
+    }
+/>
 
-## Actions on Selected Items
+In the rare case where you need to display status but cannot fit in neither an avatar nor a status stripe, such as the mini-list inside a [scorecard component](https://pxblue-components.github.io/react/?path=/info/components-score-card--get-read-me-story), you may color-code your list item icons and labels. However, for accessibility concerns, you should never rely _solely_ on the color to convey the message.
 
-<ImageGrid images={[Image6]} />
-<DemoCard repository={'multiselect-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fmultiselect-list%2Fmultiselect-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/multiselect-list/index.tsx&initialpath=multiselect-list'} ionic reactNative />
+<ImageGrid
+    images={[ScorecardImage]}
+    regularWidth
+    caption={[
+        'A scorecard with a mini-list in the card body.',
+        `1: In addition to the change of color for the icon, you can also change icons, make label text descriptive, and bolden the font.`,
+    ]}
+/>
 
-You may want to take action on several list items at once, but not all. These should be presented in a persistent snackbar (similar to a [bottom sheet](/patterns/overlay)) after selecting items. Selection can be handled in several ways, the simplest of which is to include a checkbox for each list item.
+Using tags is another way to display list status. They typically appear at the very end of a list item.
 
-## Drag and Drop
+<ImageGrid
```

Comment:
> Is this something we would actually expect someone to do in an application? 

It is stolen and sanitized from GP. Guessing from their design file, I think the status stripe is meant to highlight anything attention worthy (like your own account, people pending approves)

Reply:
It's not my favorite design, but it can slide. I would maybe split them out and show three separate examples of how a list item could indicate status (one for stripe, one for badge, one for LIT), but I like that this image has some context and shows them all at once.

Reply:
I took out the status stripe, as it is already covered by previous examples. I think the remaining ones, dotted badge works fine, with the tag here